registration-processor/registration-processor-info-storage-service/src/main/java/io/mosip/registration/processor/packet/storage/utils/AuthUtil.java

```diff
@@ -189,8 +207,10 @@ public AuthResponseDTO authByIdAuthentication(String individualId, String indivi
 		authRequestDTO.setRequestHMAC(CryptoUtil.encodeToURLSafeBase64(byteArray));
 		regProcLogger.debug(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.USERID.toString(), individualId,
 				"AuthUtil::authByIdAuthentication()::INTERNALAUTH POST service call started");
-
-		HttpEntity<AuthRequestDTO> httpEntity = new HttpEntity<>(authRequestDTO);
+		MultiValueMap<String, String> headers = new LinkedMultiValueMap<String, String>();
+		headers.add("Authorization", getToken());
```

Comment:
This header value need not to be auth token. please add some placeholder ling getConsentToken() that returns some dummy text like "sample consent token".

Reply:
Modified the code. Please check now.